Sources/MapboxMaps/Location/Puck/Puck2D.swift

```diff
@@ -190,6 +190,24 @@ public extension Puck2DConfiguration {
                                    bearingImage: showBearing ? makeBearingImage() : nil,
                                 shadowImage: UIImage(named: "location-dot-outer", in: .mapboxMaps, compatibleWith: nil)!)
     }
+    
+    var accuracyRingColor: UIColor {
+        set {
+            resolvedAccuracyRingColor = newValue
+        }
+        get {
+            return resolvedAccuracyRingColor ?? UIColor(red: 0.537, green: 0.812, blue: 0.941, alpha: 0.3)
+        }
+    }
+    
+    var accuracyRingBorderColor: UIColor {
+        set {
+            resolvedAccuracyRingBorderColor = newValue
+        }
+        get {
+            return resolvedAccuracyRingBorderColor ?? UIColor(red: 0.537, green: 0.812, blue: 0.941, alpha: 0.3)
+        }
+    }
 }
```

Comment:
Make these public stores properties in the main definition of Puck2DConfiguration just below `showsAccuracyRing`